README.md

```diff
@@ -55,6 +55,7 @@ Supported keys of the map are:
 ## License
 
 Copyright © 2016 Coda Hale
+Copyright © 2017 Sentry
```

Comment:
If Sentry wants to take over this project, I'm happy to talk about that, but this is weird.